.travis.yml

```diff
@@ -30,6 +30,7 @@ env:
   - DOCKER=docker
   - TRAVIS_TIMEOUT=240
   - TAG=0.13.0-rc.1
+  - BUSTEDVERSION=2.0.rc12
```

Comment:
looking at the above constants the convention is to split the words with `_` so `BUSTED_VERSION`